fastly/fixtures/wasm_package/get.yaml

```diff
@@ -0,0 +1,48 @@
+---
+version: 1
+interactions:
+- request:
+    body: ""
+    form: {}
+    headers:
+      User-Agent:
+      - FastlyGo/1.15.0 (+github.com/fastly/go-fastly; go1.14.3)
+    url: https://api.fastly.com/service/2Q0B1WlDbii9DItrMCvIGL/version/2/package
```

Comment:
I'm not sure how we want to handle this.

For VCL services, we have this `fixFixtures` script that sets some default serviceID.

https://github.com/fastly/go-fastly/blob/master/scripts/fixFixtures.sh

Reply:
Could you provide me with an appropriate default Wasm serviceID (at a guess a Wasm service in a fastly account?)... then I'll be able to update fixFixtures to replace both the Wasm and VCL serviceID.